packages/libffi/CONFIG

```diff
@@ -0,0 +1,25 @@
+libffi_VERSION ?= 3.3-rc0
+
+TAR_EXTENSION = .tar.gz
+
+PATCHES_3.3-rc0 = \
+        0001-Fix-installation-location-of-libffi.patch \
+        0002-Fix-use-of-compact-eh-frames-on-MIPS.patch \
+        0003-libffi-enable-hardfloat-in-the-MIPS-assembly-code.patch
```

Comment:
(Utterly trivial, but four space indent is more consistent with usage elsewhere.)

Reply:
Do you want me to drop the last 2 patches?